src/connect/ConnectModal.tsx

```diff
@@ -209,21 +209,21 @@ const ConnectModal = ({
       }`}
     >
       <div
-        className={tw`w-96 pt-12 pb-4 px-6 bg-[#FAFAFA] relative rounded-xl shadow-lg`}
+        className={tw`w-96 pt-8 pb-4 px-6 bg-[#FAFAFA] relative rounded-xl shadow-lg`}
       >
         <h1
-          className={tw`pt-2 text-2xl font-bold ${
+          className={tw`pt-2 text-xl font-semibold ${
             success ? "text-center" : "text-left"
           } `}
         >
           {success
             ? `Welcome ${displayAddress(walletAddress as string)}`
-            : "Sign-In with Your Wallet"}
+            : "Login with your wallet"}
```

Comment:
Why the change in copy? Not opposed, but think we should be consistent with https://picketapi.com/

Reply:
I'm open to Sign In or Login. Don't think I have a strong preference... Just no dashes.

Reply:
Do you have a preference? Any reasoning behind it?

Reply:
I opt-ed for consistency with https://login.xyz/

Reply:
Let's do log in. I followed the NY Times title case convention for capitalization